.envs/.local/.caluma

```diff
@@ -12,3 +12,13 @@ MINIO_STORAGE_MEDIA_BUCKET_NAME=caluma-media
 MINIO_STORAGE_SECRET_KEY=mysagw-minio123
 MINIO_STORAGE_USE_HTTPS=true
 MINIO_DISABLE_CERT_CHECKS=true
+
+# Mailing
+EMAIL_HOST=mail
+EMAIL_PORT=1025
+MAILING_FROM_NAME=mySAGW
+MAILING_FROM_MAIL=noreply@mysagw.adfinis.com
+MAILING_SENDER = "mySAGW <noreply@mysagw.adfinis.com>"
```

Comment:
Not sure we can have spaces in env files.. Those are supposed to be shell syntax, which means no spaces allowed around the `=` sign, no?

Reply:
Right! Fixed